### DISCUSSION TITLE
How to install libshout icecast/shoutcast library using vcpkg?

### DISCUSSION THREAD

**Discussion by el07694** [microsoft/vcpkg#40162] [Q&A]
How to install libshout icecast/shoutcast library using vcpkg?

I tried cmake, vcpkg, visual studio.
Every time another error.
Only MSYS2 successed!

**Comment by Osyotr** [comment]
There's currently no port for `libshout` in vcpkg.
If you want to create one, keep in mind that the library has not been tested with MSVC toolchain for many years and requires some patching.

**Comment by el07694** [comment]
Ok, thanks for the advice.

Keep in mind to keep in touch for further update.
